build/__functions.sh

```diff
@@ -90,6 +90,41 @@ function shouldStageRuntimeVersion()
 			return 0
 		fi
 		;;
+	'python'
```

Comment:
Does there need to be a `)` after each string we're checking `$platformName` against? We do that above with

```bash
'dotnet'|'dotnetcore')
```

I would think we would want to do `python)`, `node)`, etc. for the values we're adding here.

Reply:
Yep. @waliMSFT, a good way that you can test a lot of these changes is by running the scripts to build runtime images and runtime bases locally for each language. e.g.
- `./build/buildRunTimeImages.sh dotnet buster`
- `./build/buildRunTimeImageBases.sh python bullseye`